dockers/pipectl-base/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.13
```

Comment:
how about `latest` tag 👀

Reply:
I couldn't make up my mind, but as you said, using the latest can reduce our maintenance cost. Besides, seems like alpine isn't likely to make a breaking change.

Reply:
No, do not use the latest. Keep the tools/dependencies immutable.

Reply:
We basically keep our mind that not using the latest, every time we refer to the specific version. But looks like alpine can be an exception. 

@nghialv What do you think?

Reply:
Ah, again browser refresh's stuff 😓 
Okay, I'd follow that!

Reply:
🙆‍♀️ I do not have strong opinion on that so let's make it as is 👌